ingest/expression_files/dense_ingestor.py

```diff
@@ -224,7 +246,9 @@ def check_gene_keyword(header: List, row: List):
             return True
         if DenseIngestor.is_r_formatted_file(header, row)[0]:
             return True
-        raise ValueError("Required 'GENE' header is not present")
+        msg = "Required 'GENE' header is not present"
```

Comment:
Double-quotes for user-facing text:

```suggestion
        msg = "Required \"GENE\" header is not present"
```

Reply:
`d18014a`